monkey/monkey_island/cc/server_config.json.develop

```diff
@@ -1,5 +1,8 @@
 {
-  "server_config": "password",
   "deployment": "develop",
```

Comment:
```suggestion
  "data_dir": "$HOME/.monkey_island",
```

Reply:
Since the island will supply a default data_dir, I'd prefer to leave this out. Otherwise we need a `.develop.linux` and a `.develop.windows` file to cover multiple development platforms.

Reply:
Right. This should be removed then, there's already a "deployment" in "environment".